articles/active-directory/develop/howto-add-app-roles-in-azure-ad-apps.md

```diff
@@ -173,9 +173,11 @@ Because these are _application permissions_, not delegated permissions, an admin
 
 The **Status** column should reflect that consent has been **Granted for \<tenant name\>**.
 
-## Use app roles in your web API
+## Usage scenario of app roles
```

Comment:
We shared deep links with customers and partners. Could we add a `<a>` to maintain the deep links if we rename h2s?

Reply:
Done.